File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;

class Solution {
    public int[] solution(String[] wallpaper) {
        List<Integer> x = new ArrayList<>();
        List<Integer> y = new ArrayList<>();
        for (int i = 0; i < wallpaper.length; i++) {
            for (int j = 0; j < wallpaper[0].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    x.add(i);
                    y.add(j);
                }
            }
        }
        
        int[] answer = {0, 0, 0, 0};
        answer[0] = Collections.min(x);
        answer[1] = Collections.min(y);
        answer[2] = Collections.max(x) + 1;
        answer[3] = Collections.max(y) + 1;
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    x, y = [], []
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                x.append(i)
                y.append(j)
    return [min(x), min(y), max(x) + 1, max(y) + 1]